src/components/items/AllProductsHeader.jsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from 'react-router-dom';
+import styles from './AllProductsHeader.module.css';
+import { ReactComponent as SearchIcon } from 'assets/imgs/ic_search.svg';
+import { ReactComponent as DropdownArrowDown } from 'assets/imgs/ic_arrow_down.svg';
+import { ReactComponent as SortIcon } from 'assets/imgs/ic_sort.svg';
+
+const AllProductsHeader = ({
+  responsiveAllProductCount,
+  isDropdown,
+  setIsDropdown,
+  sortProduct,
+  setSortProduct,
+}) => {
+
+  return (
+    <div className={styles['container']}>
+      <div className={styles['title-container']}>
+        <h4>전체 상품</h4>
+        <div className={styles['search-container']}>
+          <div className={styles['search-box']}>
+            <SearchIcon className={styles['search-icon']} />
+            <input className={styles['search-input']} type="text" placeholder="검색할 상품을 입력해주세요" />
+          </div>
+          <Link to='/additem'><button className={styles['search-button']}>상품 등록하기</button></Link>
```

Comment:
P2:
Link태그안에 button태그가 있기보다 아래처럼 하시는 것을 추천드립니다.

```suggestion
          <Link to='/additem' className={styles['search-button']}>상품 등록하기</Link>
```